libcxx/include/__tree

```diff
@@ -2062,10 +2062,11 @@ template <class _Tp, class _Compare, class _Allocator>
 template <class _Key>
 typename __tree<_Tp, _Compare, _Allocator>::size_type
 __tree<_Tp, _Compare, _Allocator>::__erase_unique(const _Key& __k) {
-  iterator __i = find(__k);
```

Comment:
I think we should lift this optimization into `find()` instead. To do that we probably need to introduce `__find_multi` and `__find_unique`, which will have different performance characteristics.